cli/cli.ts

```diff
@@ -450,18 +407,18 @@ function ciAsync() {
         fs.writeFileSync(npmrc, cfg)
     }
 
-    const latest = branch == "master" ? "latest" : "git-" + branch
-    // upload locs on build on master
-    const masterOrReleaseBranchRx = /^(master|v\d+\.\d+\.\d+)$/;
-    const apiStringBranchRx = pxt.appTarget.uploadApiStringsBranchRx
-        ? new RegExp(pxt.appTarget.uploadApiStringsBranchRx)
-        : masterOrReleaseBranchRx;
+    const defaultBranch = pxt.github.isDefaultBranch(branch)
+    const latest = defaultBranch ? "latest" : "git-" + branch
+    // upload locs on build on default or releases
+    const apiStringBranchRx: (t: string) => boolean = ((t:string) => new RegExp(pxt.appTarget.uploadApiStringsBranchRx).test(t))
```

Comment:
```suggestion
    const apiStringBranchRx: (t: string) => boolean = ((t: string) => new RegExp(pxt.appTarget.uploadApiStringsBranchRx).test(t))
```

to make the linter happen